pathways/consignments.py

```diff
@@ -542,20 +561,20 @@ def add_contaminant_clusters_to_boxes(config, consignment):
     cluster_sizes = _contaminated_boxes_to_cluster_sizes(
         contaminated_boxes, max_contaminated_units_per_cluster
     )
-    num_strata, cluster_strata = create_stratas_for_clusters(
+    cluster_strata = choose_strata_for_clusters(
         num_boxes, max_contaminated_units_per_cluster, cluster_sizes
     )
     # Contaminate full boxes in all clusters except the last one
     for index, cluster_size in enumerate(cluster_sizes[:-1]):
-        cluster_start = math.floor(num_boxes / num_strata) * cluster_strata[index]
+        cluster_start = max_contaminated_units_per_cluster * cluster_strata[index]
```

Comment:
No, this line just computes the first index of the strata, which will always be the cluster width * the strata index. (e.g., 10 boxes, divided into 5 strata with 2 boxes each. The index of the first box of the third strata is 4.) Let me know if that still seems off.

Reply:
I guess the _cluster width_ part is what is worrying me. To clarify, I'm concerned about sanitizing user input, not about the purely theoretical equation. (Although the comments is helpful!) What if user sets `clustered/max_contaminated_units_per_cluster` to a million?

Reply:
Oh I see... I can either add min(max_contaminated_units_per_cluster, num_units) right after max_contaminated_units_per_cluster is defined or a valueerror that prompts the user to lower the cluster width. Which do you think is better? I'll do the same for the add_contaminant_clusters_to_items function. I can make sure we have this check for the other user defined variables as well.

Reply:
Since it is called `max...` I would say `min()` is the right solution since user just says it should be lower than that which still holds after calling `min()`. However, this seems like a different case as it is used as cluster width. Maybe I missed the point or the terminology here.

Reply:
`max_...` is misleading I think. Really `max_contamined_units_per_cluster` is the number of contaminated units in a cluster, which is the cluster width if contamination unit is boxes, or if using continuous distribution with item contamination unit. I have changed the variable name and documentation to be more clear. (See latest push) With that in mind, do you think it is still better to automatically adjust it, or should we let the user know that the value they put in the config is too high? And the same question applies more generally if I find other user defined variables that need the same check applied.

Reply:
Now it makes sense and the adjustment is not needed. Wrong value should result in an error.